app/views/vouchers/created.html.erb

```diff
@@ -17,6 +17,21 @@
   <%= render @voucher %>
 </section>
 
+<section class="mt-3 px-6">
+  <h2 class="text-7xl font-bold text-center">Send Digital Voucher</h2>
+  <div class="space-x-4 text-center">
+        <%= form_for(@voucher, url: send_voucher_path(@voucher.id), method: :post) do |form| %>
+              <div class="inline-block p-6">
+                <%= form.submit "E-mail", class: "p-2 bg-blue-600 text-5xl text-white hover:bg-blue-900 cursor-pointer" %>
```

Comment:
won't these buttons reload the page since they're `type="submit"` ?

Reply:
They don't reload at the moment.